timezone/src/modules/Timezone.rb

```diff
@@ -984,6 +1000,26 @@ def Summary
       HTML.List(ret)
     end
 
+    # Determines whether timezone is read-only for the current product
+    #
+    # @return [Boolean] true if it's read-only; false otherwise.
+    def readonly
+      @readonly ||= ProductFeatures.GetBooleanFeature("globals", "readonly_timezone")
+    end
+
+    # Product's default timezone when it's not defined in the control file.
+    FALLBACK_PRODUCT_DEFAULT_TIMEZONE = "UTC"
+
+    # Determines the default timezone for the current product
+    #
+    # If not timezone is set, "UTC" will be used.
```

Comment:
I introduced the constant after writing the documentation and I forgot to change it.